packages/viewer/src/components/viewer-view-cube/readme.md

```diff
@@ -0,0 +1,17 @@
+# vertex-viewer-view-cube
+
+
+
+<!-- Auto Generated Below -->
+
+
+## Properties
+
+| Property | Attribute | Description | Type                      | Default     |
+| -------- | --------- | ----------- | ------------------------- | ----------- |
+| `viewer` | --        |             | `HTMLVertexViewerElement` | `undefined` |
```

Comment:
do we want to make sure this has some useful content? or not important right now?

Reply:
I would probably be of the opinion that it's not important right now (our dev portal currently only has the README for the viewer component). Certainly open to feedback on that, but I'm thinking the documentation would be good to update once this view cube is fully functional

Reply:
makes sense to me. will approve when it's ready to be switched over to master.